pkg/handlers/app.go

```diff
@@ -167,18 +167,24 @@ func responseAppFromApp(a *apptypes.App) (*types.ResponseApp, error) {
 		return nil, errors.Wrap(err, "failed to get license")
 	}
 
-	isIdentityServiceSupportedForVersion, err := store.GetStore().IsIdentityServiceSupportedForVersion(a.ID, a.CurrentSequence)
+	appVersions, err := store.GetStore().FindAppVersions(a.ID)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to check if identity service is supported for version %d", a.CurrentSequence)
+		return nil, errors.Wrap(err, "failed to get app versions")
+	}
+	topVersionSequence := appVersions.AllVersions[0].ParentSequence
```

Comment:
`topVersionSequence` can be confusing. how about:
```
latestVersion := appVersions.AllVersions[0]
```
and then use `latestVersion.ParentSequence`?